app/views/articles/index.html.erb

```diff
@@ -5,18 +5,20 @@
     </div>
     <% if @articles.any? %>
       <% @articles.each_with_index do |article, article_num| %>
-        <div class="page-article">
-          <p class="article-title"><span class="article-id"><%= article_num + 1 %>. </span><%= article.title %>
-            <%= link_to 'Show', article_path(article), {class: 'article-link'} %> /
-            <%= link_to 'Delete', article_path(article), method: :delete, data: {confirm: 'Are you sure?'}, class: "article-link" %>
-          </p>
-          <p class="article-body"><%= strip_tags(article.body)[0..160].gsub(/\s\w+\s*$/,'...') %></p>
-        </div>
-      <% end %>
-    <% else %>
       <div class="page-article">
-        <p class="article-title">Nothing found...</p>
+        <p class="article-title"><span class="article-id"><%= article_num + 1 %>. </span><%= article.title %>
+          <%= link_to 'Show', article_path(article), {class: 'article-link'} %> /
+          <%= link_to 'Delete', article_path(article), method: :delete, data: {confirm: 'Are you sure?'}, class: "article-link" %>
+        </p>
+        <p class="article-body">
+          <%= strip_tags(article.body)[0..160].gsub(/\s\w+\s*$/,'...') %>
```

Comment:
There's something going on here but at first glance I have no clue.
Is there a named method you could put this into for the `Article` model?
like, `preview_body`?

This will be easier to test too.